### PR TITLE
refactor(formatters): refactor formatters.js

### PR DESCRIPTION
**Pull request by lusarz**
Refactor `lib/init/formatters` to use ES6

### CODE REVIEW

File: config.tpl.coffee [hunk 1]
```diff
@@ -25,7 +25,8 @@ module.exports = (config) ->
 
     # preprocess matching files before serving them to the browser
     # available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
-    preprocessors: %PREPROCESSORS%
+    preprocessors: {%PREPROCESSORS%
+    }
```
Comment (dignifiedquire) [inline config.tpl.coffee]: why this change?

File: lib/utils/file-utils.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const fs = require('graceful-fs')
```
Comment (dignifiedquire) [inline lib/utils/file-utils.js]: please add `'use strict'`

File: config.tpl.coffee (hunk 1 above)
Comment (lusarz) [reply config.tpl.coffee]: to be more consistent with way `files`, `exclude` are built.

File: lib/utils/file-utils.js (hunk 2 above)
Comment (lusarz) [reply lib/utils/file-utils.js]: Done
